css/style.css

```diff
@@ -1,19 +1,100 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* header (nav) */
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+.nav_bar {
+  background-color: white;
+  position: fixed;
+  width: 100%;
+  height: 4.5rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  top: 0;
+}
+
+.nav_logo {
+  height: 2.2em;
+  padding-top: 0.5em;
+  margin-left: 7em;
+}
+
+.nav_list {
+  font-size: 1rem;
+}
+
+.nav_list li {
+  display: inline-block;
+  margin-left: 1rem;
+  color: grey;
+}
+
+.nav_list .highlighted {
+  font-weight: 500;
+  color: rgb(75, 73, 73);
+}
+/* intro main */
+
+.intro {
+  background-image: url("../img/first-background.jpg");
+  background-position: center;
+  height: 45rem;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+.intro h1 {
+  color: white;
+  text-align: center;
+  padding-top: 10rem;
+  font-size: 3rem;
+  font-weight: 300;
+}
+
+.intro h3 {
+  color: white;
+  text-align: center;
+  font-size: 1.5rem;
+  font-weight: 300;
+}
 
+/* Button */
 
+.button {
+  text-align: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 1.125rem;
+  background-color: orange;
+  border-color: orange;
+  border-radius: 4px;
+  padding: 1rem 1.5rem;
+}
+
+/* Features */
+
+.devices {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.features h2 {
+  font-size: 2.5rem;
+  font-weight: 300;
+}
+
+.feature_title {
+  display: flex;
+  justify-content: center;
+}
+
+.caption {
+  font-size: 1.1rem;
+  justify-content: center;
+  font-weight: 2;
```

Comment:
Is font weight 2 a property? Is 500, 700 or bold or regular an option? https://www.w3schools.com/cssref/pr_font_weight.asp